src/asset-picker/AssetPickerDropDown.js

```diff
@@ -35,26 +35,27 @@ export default class AssetPickerDropDown extends Component {
             router.push('asset-picker');
         }
     }
+    onClose(e) {
+        const { actions } = this.props;
+        e.stopPropagation();
```

Comment:
Our drop is on a a modal with some higher z-index and drown above other controls. Closing it involved clicking any part of the page away from the dropdown . Wouldn't b a good practice to stop propagation in case of any which in this case is possible? :wink: